packages/stark-ui/src/modules/dropdown/components/dropdown.component.html

```diff
@@ -0,0 +1,65 @@
+<mat-form-field class="stark-dropdown"> <!--valdr-form-group-->
```

Comment:
Please remove the comment about `valdr-form-group`... we have it on the roadmap ;)